core/src/components/datetime/datetime.scss

```diff
@@ -383,10 +382,6 @@ ion-picker-column-internal {
   z-index: -1;
 }
 
-.calendar-day-highlight:after {
```

Comment:
Since we are getting rid of the pseudo element, do we still need FW-1720 and the associated hack/workaround above?

Reply:
Good catch! Removed 👍